src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerEventExtensions.cs

```diff
@@ -49,5 +49,19 @@ public static void LogEvent(this ILogger logger, string name, Dictionary<string,
 
             logger.LogWarning(MessageFormats.EventFormat, new EventLogEntry(name, context));
         }
+
+        /// <summary>
+        /// Logs a custom event
+        /// </summary>
+        /// <param name="logger">The logger to track the telemetry.</param>
+        /// <param name="name">Name of the event</param>
+        /// <param name="context">Context that provides more insights on the event that occurred</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> is <c>null</c>.</exception>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="name"/> is blank.</exception>
+        [Obsolete("Use " + nameof(LogCustomEvent) + " instead")]
+        public static void LogEvent(this ILogger logger, string name, Dictionary<string, object> context = null)
+        {
+            LogCustomEvent(logger, name, context);
```

Comment:
```suggestion
        {
            Guard.NotNull(logger, nameof(logger), "Requires a logger instance to track telemetry");
            Guard.NotNullOrWhitespace(name, nameof(name), "Requires a non-blank event name to track an custom event");

            context = context ?? new Dictionary<string, object>();

            LogCustomEvent(logger, name, context);
```
We should not alter the implementation of obsolete fuctionality.